entry-node.js

```diff
@@ -1,84 +1,8 @@
 var library = require("./index");
 
-var locales = [
-  require("./langs/cy.json"),
-  require("./langs/dv.json"),
-  require("./langs/sw.json"),
-  require("./langs/eu.json"),
-  require("./langs/af.json"),
-  require("./langs/am.json"),
-  require("./langs/ha.json"),
-  require("./langs/ku.json"),
-  require("./langs/ml.json"),
-  require("./langs/no.json"),
-  require("./langs/ps.json"),
-  require("./langs/sd.json"),
-  require("./langs/so.json"),
-  require("./langs/sq.json"),
-  require("./langs/ta.json"),
-  require("./langs/tg.json"),
-  require("./langs/tt.json"),
-  require("./langs/ug.json"),
-  require("./langs/ur.json"),
-  require("./langs/vi.json"),
-  require("./langs/ar.json"),
-  require("./langs/az.json"),
-  require("./langs/be.json"),
-  require("./langs/bg.json"),
-  require("./langs/bn.json"),
-  require("./langs/bs.json"),
-  require("./langs/ca.json"),
-  require("./langs/cs.json"),
-  require("./langs/da.json"),
-  require("./langs/de.json"),
-  require("./langs/el.json"),
-  require("./langs/en.json"),
-  require("./langs/es.json"),
-  require("./langs/et.json"),
-  require("./langs/fa.json"),
-  require("./langs/fi.json"),
-  require("./langs/fr.json"),
-  require("./langs/gl.json"),
-  require("./langs/el.json"),
-  require("./langs/he.json"),
-  require("./langs/hi.json"),
-  require("./langs/hr.json"),
-  require("./langs/hu.json"),
-  require("./langs/hy.json"),
-  require("./langs/id.json"),
-  require("./langs/is.json"),
-  require("./langs/it.json"),
-  require("./langs/ja.json"),
-  require("./langs/ka.json"),
-  require("./langs/kk.json"),
-  require("./langs/km.json"),
-  require("./langs/ko.json"),
-  require("./langs/ky.json"),
-  require("./langs/lt.json"),
-  require("./langs/lv.json"),
-  require("./langs/mk.json"),
-  require("./langs/mn.json"),
-  require("./langs/ms.json"),
-  require("./langs/nb.json"),
-  require("./langs/nl.json"),
-  require("./langs/nn.json"),
-  require("./langs/pl.json"),
-  require("./langs/pt.json"),
-  require("./langs/ro.json"),
-  require("./langs/ru.json"),
-  require("./langs/sk.json"),
-  require("./langs/sl.json"),
-  require("./langs/sr.json"),
-  require("./langs/sv.json"),
-  require("./langs/th.json"),
-  require("./langs/tr.json"),
-  require("./langs/uk.json"),
-  require("./langs/uz.json"),
-  require("./langs/zh.json"),
-];
-
-for (var i = 0; i < locales.length; i++) {
-  library.registerLocale(locales[i]);
+for (var i = 0; i < library.supportedLocales.length; i++) {
```

Comment:
could we use library.getSupportedLanguages() instead of library.supportedLocales?